common/lib/xmodule/xmodule/modulestore/tests/utils.py

```diff
@@ -255,19 +255,19 @@ def build(self, **kwargs):
         """
         contentstore = kwargs.pop('contentstore', None)
         if not contentstore:
-            with self.build_without_contentstore() as (contentstore, modulestore):
+            with self.build_without_contentstore(**kwargs) as (contentstore, modulestore):
```

Comment:
yes - exactly.  This test runs just under nosetools on Jenkins.
And as we both know, running with python manage.py adds much more overhead.  So unless absolutely necessary, I thought about keeping it as a non-django test.